tests/k8s_test.go

```diff
@@ -0,0 +1,113 @@
+// +build: k8s
```

Comment:
// +build k8s

Reply:
ok, fixed in `tests/k8s: add tests for k8s graph.Node creation`